packages/route/config/doctrine/Route/Route.orm.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+    <entity
+        name="Sulu\Route\Domain\Model\Route"
+        table="ro_routes"
+    >
+        <unique-constraints>
+            <unique-constraint name="ro_routes_unique" fields="site,locale,slug" />
+            <unique-constraint name="ro_routes_resource_unique" fields="site,locale,resourceKey,resourceId" />
+        </unique-constraints>
+
+        <id name="id" type="integer" column="id">
+            <generator strategy="AUTO"/>
+        </id>
+
+        <field name="site" type="string" length="16" nullable="true"/>
+        <field name="locale" type="string" length="11"/>
+        <field name="slug" type="string"/>
```

Comment:
To support or versions of mysql site + locale + slug can not be longer together as `191` chars. Which means in this case 191 - 16 -11 = 164 chars for the slug.

There would be one workaround to support longer versions of URL by using a additional `slugHash` field based on widely supported `SHA1` algorithm. That way we can support longer Urls 164.

@benr77 already requested that the current `title` of 64 is to small for his usecases. If we increase that to 128 or 191. A longer parent title and a child page or 3 levels of navigation could already hit the 191 limit and we would require trim the url correctly. But that is more a theoretical issue. Which may already existed before.

Reply:
Interesting the workaround with a `Hash` is also documented in MySQL docs itself but as a alternative for composite indexes so n our case it would include the site, locale, slug: https://dev.mysql.com/doc/refman/8.4/en/multiple-column-indexes.html